app/views/admin/configs/show.html.erb

```diff
@@ -243,6 +243,15 @@
                                placeholder: "We're a place where coders share, stay up-to-date and grow their careers." %>
               <div class="alert alert-info">Used in signup modal.</div>
             </div>
+
+            <div class="form-group">
+              <%= admin_config_label :community_copyright_start_year %>
+              <%= f.text_field :community_copyright_start_year,
+                               class: "form-control",
+                               value: SiteConfig.community_copyright_start_year,
+                               placeholder: "2020" %>
+              <div class="alert alert-info">Used in the footer to note what year the forem started.</div>
```

Comment:
It looks like some of the optimizations using constants in this file either got reverted or just not committed on master at some point 🤷🏽‍♀️, but I'll look into this it in another PR.  

Anyways, you'll see in some parts of this file, e.g. https://github.com/forem/forem/blob/cc021307438a126b96d55fba2cb38207f87ec34b/app/views/admin/configs/show.html.erb#L598-L603 we're moving towards using Constants for the description and placeholder, which looks cleaner and makes it easier to reuse the descriptions across sections as we progress. 

So, in this case we would want to change the description to `Constants::SiteConfig::DETAILS[config_key][:description]` and placeholder to `Constants::SiteConfig::DETAILS[config_key][:placeholder]` and add it to this file https://github.com/forem/forem/blob/master/app/lib/constants/site_config.rb

Reply:
Ah that makes sense.... I added it in for this PR, and we can fix this holistically in another.